includes/core-testing-note-aspnet.md

```diff
@@ -0,0 +1,13 @@
+>[!IMPORTANT]
+>If you're _not_ using the **.NET Core 3.0 SDK**, you must add two additional package references, `Microsoft.AspNetCore.App` and `Microsoft.AspNetCore.Mvc.Testing`.
```

Comment:
You will need to add the Microsoft.AspNetCore.App if TargetFramework is less than netcoreapp3.0, even if you are building with the 3.0 SDK.